### PR TITLE
auxtimer, change

### PR DESCRIPTION
**Pull request by 7omas**
Added to match new feature in Ontime 3.9.2

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -456,7 +456,8 @@
             "Set": "set",
             "Start": "start",
             "Pause": "pause",
-            "Stop": "stop"
+            "Stop": "stop",
+            "Change": "change"
```
Comment (alex-Arc) [inline module.json]: Could we call it add/remove
to keep inline with the Ontime naming?

Comment (7omas) [reply module.json]: Offset is currently used in the main timer. Is that an option or should both places instead be 'add/remove' and call the internal variable addtime to match the API call?

Comment (alex-Arc) [reply module.json]: hmm yeah I missed that
it would be good to have them the same I think

**Review by alex-Arc — APPROVED** [review]
Nice 🚀

**Pull request merged**